opencart 1.5.6.4/upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -114,6 +129,64 @@
             </tr>  
           </table>
         </div>
+        <!-- Customer sync -->
+        <div id="tab-sync">
+          <table class="form">
+            <tr>
+              <td><?php echo $entry_enable_subscriber_title; ?></td>
+              <td><select name="smaily_for_opencart_enable_subscribe" id="input-subscriber-status" class="form-control">
```

Comment:
Formatting: multiline (does not have closing tag on the same line) HTML element should start on a new line.